Makefile

```diff
@@ -474,14 +474,6 @@ test.istio: gotestsum
 		-timeout $(E2E_TEST_TIMEOUT) \
 		./test/e2e/...
 
-.PHONY: test.expression_router
-test.expression_router: gotestsum
```

Comment:
I'm surprised that wasn't included anywhere in the CI.

Reply:
Yeah, apparently we missed that. 😿